BinaryEncoding.md

```diff
@@ -451,6 +451,8 @@ be skipped over by an engine. The current list of valid `name_type` codes are:
 | --------- | ---- | ----------- |
 | [Function](#function-names) | `1` | Assigns names to functions |
 | [Local](#local-names) | `2` | Assigns names to locals in functions |
+| [Module](#module-name) | `3` | Assigns a name to the module |
```

Comment:
Could we make the code be 0 so that it looks like we meant to do this all along ;) ?